nbs/test_viz.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
so there was no valuable wisdom in this notebook that was worth preserving? :-)

Reply:
As I said before, the file is too large with plotly plots

Reply:
then why not just strip the output? :-)

Reply:
Just click "clear output" button in vscode ...